modules/users/client/views/authentication/signin.client.view.html

```diff
@@ -49,42 +49,40 @@
                   </span>
                 </div>
               </div>
-              <a ui-sref="forgot({ userhandle: auth.credentials.username })"
-                 class="btn btn-inverse"
-                 ng-class="{ 'btn-lg btn-block': auth.authError }"
-                 tabindex="4">
-                Forgot?
-                <span ng-if="auth.authError" class="hidden-xs">
-                  Recover your password
-                </span>
-              </a>
-              <div class="text-center form-group">
-                <br><br>
+              <div class="form-group">
                 <button type="submit"
-                        class="btn btn-default btn-action"
+                        class="btn btn-default btn-lg"
                         ng-disabled="auth.isLoading"
                         tabindex="3">
                   <span ng-if="!auth.isLoading">
                     {{ auth.continue ? 'Sign in to continue' : 'Login' }}
                   </span>
                   <span ng-if="auth.isLoading">Wait...</span>
                 </button>
+                <span class="btn space-h"></span>
+                <a ui-sref="forgot({ userhandle: auth.credentials.username })"
+                   class="btn-link"
+                   ng-class="{ 'btn-lg btn-link': auth.authError }"
+                   tabindex="4">
+                  Forgot?
+                  <span ng-if="auth.authError" class="hidden-xs">
+                    Recover your password
+                  </span>
+                </a>
               </div>
             </fieldset>
           </form>
+          <div class="text-white">
```

Comment:
it is missing hidden-xs class so it is displayed twice in the mobile view
![selection_007](https://user-images.githubusercontent.com/38785076/49760489-e9236280-fcc4-11e8-9dc2-aa5cd04fcad6.png)

Reply:
Also the comment `<!-- For bigger screens -->` should be before this div